ooiservices/app/uframe/controller.py

```diff
@@ -263,79 +222,72 @@ def get_uframe_stream(mooring, platform, instrument, stream):
-        if response.status_code == 200:
-            return jsonify(metadata=response.json()), 200
-        return jsonify(metadata={}), 200
+        response = requests.get("/".join([UFRAME_DATA,mooring, platform, instrument, stream, 'metadata']))
+        return response
     except:
         return internal_server_error('uframe connection cannot be made.')
 
-
-@auth.login_required
-@api.route('/get_metadata_times/<string:stream>/<string:ref>', methods=['GET'])
+#@auth.login_required
+@api.route('/get_metadata_times/<string:ref>', methods=['GET'])
 @cache.memoize(timeout=3600)
-def get_uframe_stream_metadata_times(stream,ref):
+def get_uframe_stream_metadata_times(ref):
     '''
     Returns the uFrame time bounds response for a given stream
     '''
     mooring, platform, instrument = ref.split('-', 2)
-    stream_type, stream = stream.split('_', 1)
     try:
         UFRAME_DATA = current_app.config['UFRAME_URL'] + current_app.config['UFRAME_URL_BASE']
         response = requests.get("/".join([UFRAME_DATA, mooring, platform, instrument, 'metadata','times']))
         if response.status_code == 200:
-            return jsonify(times=response.json()), 200
+            return response
         return jsonify(times={}), 200
     except:
         return internal_server_error('uframe connection cannot be made.')
 
+#@cache.memoize(timeout=3600)
+#def get_uframe_stream_contents(mooring, platform, instrument, stream_type, stream):
+#    '''
+#    Gets the stream contents
+#    '''
+#    try:
+#        UFRAME_DATA = current_app.config['UFRAME_URL'] + current_app.config['UFRAME_URL_BASE']
+#        response =  requests.get("/".join([UFRAME_DATA,mooring, platform, instrument, stream_type, stream]))
+#        if response.status_code != 200:
+#            #print response.text
+#            pass
+#        return response
+#    except:
+#        return internal_server_error('uframe connection cannot be made.')
 
 @cache.memoize(timeout=3600)
-def get_uframe_stream_contents(mooring, platform, instrument, stream_type,
-                               stream):
-    '''
-    Gets the stream contents
-    '''
-    try:
-        UFRAME_DATA = (current_app.config['UFRAME_URL'] +
-                       current_app.config['UFRAME_URL_BASE'])
-        url = '/'.join([UFRAME_DATA, mooring, platform, instrument, stream_type,
-                        stream])
-        return requests.get(url)
-    except:
-        return internal_server_error('uframe connection cannot be made.')
-
-
-@cache.memoize(timeout=3600)
-def get_uframe_stream_contents_bounded(mooring, platform, instrument, stream_type, stream, start_time, end_time):
+def get_uframe_stream_contents(mooring, platform, instrument, stream_type, stream, start_time, end_time, dpa_flag):
     '''
     Gets the bounded stream contents, start_time and end_time need to be datetime objects
     '''
-    try:
-        query = '?beginDT=%s&endDT=%s' % (start_time, end_time)
+    try:        
+        if dpa_flag == '0':
```

Comment:
We ahve to set this to a toggle.. set as 0 for now so that we can test.  CAREFUL!!